jmclient/jmclient/wallet_service.py

```diff
@@ -715,7 +715,7 @@ def sync_addresses(self):
         import_needed = self.bci.import_addresses_if_needed(addresses,
             wallet_name)
         if import_needed:
-            self.display_rescan_message_and_system_exit(self.restart_callback)
+            #self.display_rescan_message_and_system_exit(self.restart_callback)
```

Comment:
This needs some kind of rework, we can't remove it for all clients of the service.